### PR TITLE
Feature/#60

### LINKED ISSUE #60: Create landing page

### PR DESCRIPTION
**Pull request by altarin**
closes #60

### CODE REVIEW

File: makeup/html/main.css [hunk 1]
```diff
@@ -1,155 +1,464 @@
...
 }
 
-#close-btn:active, #close-btn:focus {
-    color: #ffffff;
+/*размер изображения логотипа*/
+.shop-window .blok1 img {
+    max-width: 70%;
+    max-height: 70%;
+    margin-top: 20px;
+    /*min-width: 50%;*/
```
Comment (IvanAlekseevichPopov) [inline makeup/html/main.css]: Please, remove commented line

File: makeup/html/main.html [hunk 2]
```diff
@@ -4,141 +4,562 @@
...
-                </h1>
-                <p class="lead my-2">и возвращайте до 90% от стоимости</p>
+            <!--каталог магазинов-->
+            <div class="title shops">КAТАЛОГ МАГАЗИНОВ</div>
+            <div class="shop-block">
+
+                <div class="square">
+                    <div class="shop-window">
+                        <div class="item">
+                            <a href="//">
```
Comment (IvanAlekseevichPopov) [inline makeup/html/main.html]: Is it correct link?

**Review by IvanAlekseevichPopov — COMMENTED** [review]
Please remove ngrok binary file from project root. It should not be in the project

File: makeup/html/main.css (hunk 1 above)
Comment (altarin) [reply makeup/html/main.css]: fixed

File: makeup/html/main.html (hunk 2 above)
Comment (altarin) [reply makeup/html/main.html]: fixed

**Review by IvanAlekseevichPopov — APPROVED** [review]

**Pull request merged**